recipes/sdsl-lite/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+from conans import ConanFile, tools
+
+
+class SDSLLite(ConanFile):
+    name = "sdsl-lite"
+    description = "SDSL - Succinct Data Structure Library"
+    homepage = "https://github.com/xxsds/sdsl-lite"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    topics = ("conan", "sdsl", "succint", "data-structures")
+    settings = "compiler"
+    exports = ["patches/*"]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        source = self.conan_data["sources"][self.version]
+        tools.get(**source)
+        extracted_folder = self.name + "-" + \
+            os.path.splitext(os.path.basename(source["url"]))[0]
+        os.rename(extracted_folder, self._source_subfolder)
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
```

Reply:
my company uses 1.32.1 conan, so i'd like to do this in a next recipe hash